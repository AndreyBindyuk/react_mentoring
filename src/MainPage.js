import React from 'react';
import SearchContainer from "../searchImpl/SearchContainer";
import MovieItemList from '../movieItemImpl/MovieItemList';
import Footer  from './frameImpl/Footer';

class MainPage extends React.Component{
    render(){
        return(
            <div>
            <SearchContainer />
            <MovieItemList />
            <Footer />
            </div>
        );
    }
}

export default MainPage;