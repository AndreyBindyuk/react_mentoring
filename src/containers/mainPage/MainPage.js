import React from "react";
import SearchContainer from "../searchContainer/SearchContainer";
import MovieItemList from "../movieList/MovieItemList";
import Footer from "../../shared/footer/Footer";

class MainPage extends React.Component {
  render() {
    return (
      <div>
        <SearchContainer />
        <MovieItemList />
        <Footer />
      </div>
    );
  }
}

export default MainPage;
