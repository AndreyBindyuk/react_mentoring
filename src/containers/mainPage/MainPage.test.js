import React from 'react';
import { shallow } from 'enzyme';
import MainPage  from './MainPage';

describe('<MainPage />', () => {
    it('render internal components correctly', () => {  
        const ModalTriggerComponent = shallow(<MainPage />);
        expect(ModalTriggerComponent).toMatchSnapshot();
    });
});