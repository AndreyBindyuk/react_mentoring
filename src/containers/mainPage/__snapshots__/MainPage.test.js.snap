// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage /> render internal components correctly 1`] = `ShallowWrapper {}`;
